api/sessions/option.gen.go

```diff
@@ -86,3 +87,15 @@ func WithRecursive(recurse bool) Option {
 		o.withRecursive = true
 	}
 }
+
+func WithIncludeTerminated(inIncludeTerminated bool) Option {
+	return func(o *options) {
+		o.queryMap["include_terminated"] = fmt.Sprintf("%v", inIncludeTerminated)
```

Comment:
Nit: I would prefer to use `strconv.FormatBool` here to keep the strong typing

Reply:
This is generated code

Reply:
🙄 so it is, my bad!